cluster/src/main/scala/hmda/cluster/HmdaPlatform.scala

```diff
@@ -107,11 +107,11 @@ object HmdaPlatform extends App {
 
     (supervisorProxy ? FindActorByName(SingleLarValidation.name))
       .mapTo[ActorRef]
-      .map(a => log.info(s"Started single lar validator at ${a.path}"))
+      .map(a => log.debug(s"Started single lar validator at ${a.path}"))
 
     (supervisorProxy ? FindActorByName(InstitutionPersistence.name))
       .mapTo[ActorRef]
-      .map(a => log.info(s"Started institutions at ${a.path}"))
+      .map(a => log.debug(s"Started institutions at ${a.path}"))
   }
```

Comment:
I would actually like to maintain these two as `INFO`, since they happen only once per application run and are proof that the `ActorSystem` and the main actors have started up correctly